CMakeLists.txt

```diff
@@ -79,26 +82,20 @@ add_subdirectory(src)
 # ---------------------------------------------------------------------------
 # mini Apps
 # ---------------------------------------------------------------------------
-set(DLAF_BUILD_MINIAPPS ON CACHE BOOL "Build miniapps")
 if (DLAF_BUILD_MINIAPPS)
   add_subdirectory(miniapp)
 endif()
 
 # ---------------------------------------------------------------------------
 # Test
 # ---------------------------------------------------------------------------
-set(DLAF_WITH_TEST ON CACHE BOOL "Enables tests")
-if (DLAF_WITH_TEST)
-  include(CTest)
-  enable_testing()
-
+if (BUILD_TESTING)
```

Comment:
Why remove the prefix here (`DLAF_`)?

Reply:
In the last months I realized that it is a CMake standard thing. Same for `BUILD_DOC` and `BUILD_SHARED_LIBS`.

I thought it was a good thing to do, in case I can revert it.

Reply:
Sorry, something weird happened with GitHub here, I replied yesterday but my reply was marked as pending and you seem not to have seen it....

I like having a prefix `DLAF_` instead of `BUILD_TESTING` / `BUILD_DOC` because if `add_subdirectory()` is used to add DLA-F as a subproject, we need to take special precautions to avoid activating our tests when the top project makes use of the same variable to activate their tests as discussed here : https://cliutils.gitlab.io/modern-cmake/chapters/testing.html .